modules/data.atmosphere/R/met2CF.csv.R

```diff
@@ -26,7 +26,7 @@
 ##'   lon = -80
 ##'   met2CF.csv(in.path,in.file,outfolder,format,lat,lon)
 ##' }
-met2CF.csv <- function(in.path, in.file, outfolder, format, lat=NULL, lon=NULL, nc_verbose = FALSE){
+met2CF.csv <- function(in.path, in.file, outfolder, format, lat=NULL, lon=NULL, nc_verbose = FALSE,...){
```

Comment:
This was one of the things on my to do list with the functions, but I can't remember specifically which met2CF functions will need additional arguments. I'll be having a followup conversation with Mike very soon about it.